contentcuration/contentcuration/frontend/channelList/views/Channel/ChannelList.vue

```diff
@@ -85,14 +77,20 @@
       };
     },
     computed: {
-      ...mapGetters('channel', ['getChannels', 'channels']),
-      ...mapState('channel', ['page']),
+      ...mapGetters('channel', ['channels']),
       listChannels() {
-        const channels = this.isStarred ? this.channels : this.getChannels(uniq(this.page.results));
+        const channels = this.channels;
         if (!channels) {
           return [];
         }
-        return channels.filter(channel => channel[this.listType] && !channel.deleted);
+        const sortFields = ['-modified'];
+        if (this.listType === ChannelListTypes.PUBLIC) {
+          sortFields.shift('-priority');
```

Comment:
I'm not aware that `shift` accepts arguments. Should this be `unshift`?

Reply:
Good catch - updated!

Reply:
Possibly related to the fact that this issue https://github.com/learningequality/studio/issues/2056 was fixed by the PR that this mostly reverted?